torch/_dynamo/source.py

```diff
@@ -458,7 +449,7 @@ def guard_source(self):
 @dataclasses.dataclass(frozen=True)
 class FSDPNNModuleSource(NNModuleSource):
     def guard_source(self):
-        return _GUARD_SOURCE_FSDP_MODULE[self.base.guard_source()]
+        return _GUARD_SOURCE_NN_MODULE[self.base.guard_source()]
```

Comment:
@wconstab are you okay with us removing this as we bring the different module kinds closer together?

Reply:
well directionally, yes. if you keep things working and move to a new impl im happy. 

ill review more closely

Reply:
Yeah, lmk, we don't necessarily have to do it this way.